tests/nodescaling/scale_test.go

```diff
@@ -0,0 +1,165 @@
+package tests
+
+import (
+	"testing"
+
+	"github.com/gruntwork-io/terratest/modules/terraform"
+	"github.com/rancher/rancher/tests/framework/clients/rancher"
+	management "github.com/rancher/rancher/tests/framework/clients/rancher/generated/management/v3"
+	"github.com/rancher/rancher/tests/framework/extensions/users"
+	password "github.com/rancher/rancher/tests/framework/extensions/users/passwordgenerator"
+	ranchFrame "github.com/rancher/rancher/tests/framework/pkg/config"
+	namegen "github.com/rancher/rancher/tests/framework/pkg/namegenerator"
+	"github.com/rancher/rancher/tests/framework/pkg/session"
+	"github.com/rancher/tfp-automation/config"
+	"github.com/rancher/tfp-automation/framework"
+	cleanup "github.com/rancher/tfp-automation/framework/cleanup"
+	"github.com/rancher/tfp-automation/tests/extensions/provisioning"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+)
+
+type ScaleTestSuite struct {
+	suite.Suite
+	client             *rancher.Client
+	standardUserClient *rancher.Client
+	session            *session.Session
+	terraformConfig    *config.TerraformConfig
+	clusterConfig      *config.TerratestConfig
+	terraformOptions   *terraform.Options
+}
+
+func (r *ScaleTestSuite) SetupSuite() {
+	testSession := session.NewSession()
+	r.session = testSession
+
+	client, err := rancher.NewClient("", testSession)
+	require.NoError(r.T(), err)
+
+	r.client = client
+
+	enabled := true
+	var testuser = namegen.AppendRandomString("testuser-")
+	var testpassword = password.GenerateUserPassword("testpass-")
```

Comment:
nit; is this password logged somewhere? Just checking, if we need to login as that user to debug something, can we do that as is?